frame/lending/src/tests.rs

```diff
@@ -269,22 +270,15 @@ fn accrue_interest_plotter() {
 #[test]
 fn test_borrow_repay_in_same_block() {
 	new_test_ext().execute_with(|| {
-		let collateral_amount = 900000;
+		let collateral_amount = 900000000;
 		let (market, vault) = create_simple_market();
-		// Balance for ALICE
-		assert_eq!(Tokens::balance(MockCurrencyId::USDT, &ALICE), 0);
 		assert_ok!(Tokens::mint_into(MockCurrencyId::USDT, &ALICE, collateral_amount));
-		assert_eq!(Tokens::balance(MockCurrencyId::USDT, &ALICE), collateral_amount);
 
 		assert_ok!(Lending::deposit_collateral_internal(&market, &ALICE, collateral_amount));
 		assert_eq!(Tokens::balance(MockCurrencyId::USDT, &ALICE), 0);
```

Comment:
if we go with not testing other pallets approach should we remove this line too?